src/components/RecentReviewCards.jsx

```diff
@@ -0,0 +1,132 @@
+'use client';
+
+import { useState } from 'react';
+import { decode } from 'he';
+import { Star, ArrowDown } from 'lucide-react';
+import Link from 'next/link';
+import Masonry from 'react-masonry-css';
+import styles from '@/styles/RecentReviewCards.module.css';
+import { motion } from 'framer-motion';
+import { formatDate } from '@/lib/utils';
+import useRecentReviewCards from '@/hooks/useRecentReviewCards';
+import useWindowWidth from '@/hooks/useWindowWidth';
+
+/**
+ * RecentReviewCards component displays a grid of recent reviews.
+ * It fetches reviews from a custom hook and displays them in a responsive masonry layout.
+ * 
+ * @returns {JSX.Element} The rendered RecentReviewCards component.
+ */
+export default function RecentReviewCards() {
+    // State to manage expanded reviews
+    const [expandedReviews, setExpandedReviews] = useState({});
+
+    // Custom hooks to get window width and recent reviews
+    const windowWidth = useWindowWidth();
+    const reviews = useRecentReviewCards();
+
+    // Define the breakpoint columns for the masonry layout
+    const breakpointColumnsObj = {
+        default: 3,
+        1024: 2,
+        640: 1,
+    };
+
+    // If there are no reviews, display a message
+    if (reviews.length === 0) {
+        return (
+            <div className="w-full px-4 py-10 bg-white">
+                <div className="max-w-6xl mx-auto text-center">
+                    <h2 className="text-2xl font-bold mb-4">What people are saying</h2>
+                    <p className="text-gray-600">No reviews available just yet. Be the first to leave one!</p>
+                </div>
+            </div>
+        );
+    }
+
+    return (
+        <div className="max-w-6xl mx-auto">
+            {/* Section title */}
+            {/* TODO: change to quirky header */}
+            <h2 className="text-2xl font-bold mb-6">What people are saying</h2>
```

Comment:
I'm still not sure what we said before!! Something like
```suggestion
            <h2 className="text-2xl font-bold mb-6">See whose thoughts were activated lately</h2>
```

Reply:
Not sure, I dont think it was this